internal/cli/iam/organizations/describe_test.go

```diff
@@ -18,29 +18,48 @@
 package organizations
 
 import (
+	"bytes"
 	"testing"
 
 	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
 	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
-	atlasv2 "go.mongodb.org/atlas/api/v1alpha"
+	"github.com/stretchr/testify/assert"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 func TestDescribe_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockStore := mocks.NewMockOrganizationDescriber(ctrl)
 	defer ctrl.Finish()
+	stringVal := "test"
+	expected := &atlasv2.Organization{
+		Links: nil,
+		Id:    &stringVal,
+		Name:  stringVal,
+	}
+	buf := new(bytes.Buffer)
 
 	mockStore.
 		EXPECT().
 		Organization(gomock.Eq("5a0a1e7e0f2912c554080adc")).
-		Return(&atlasv2.Organization{}, nil).
+		Return(expected, nil).
 		Times(1)
 
 	opts := &DescribeOpts{
 		store: mockStore,
 		id:    "5a0a1e7e0f2912c554080adc",
+		OutputOpts: cli.OutputOpts{
+			Template:  describeTemplateCloud,
+			OutWriter: buf,
+		},
 	}
+
 	if err := opts.Run(); err != nil {
 		t.Fatalf("Run() unexpected error: %v", err)
 	}
+	assert.Equal(t, `ID     NAME
```

Comment:
IMHO this will be making tests to flaky. Would ignore assertions.  or use contains instead

Reply:
Agree, this won't scale well as we migrate more stores and have to amend more templates